design-proposals/README.md

```diff
@@ -102,3 +102,4 @@ Next KLIP number: **62**
 | [KLIP-59: JSON functions](klip-59-json-functions.md)                                                                             | Approved |                   |            | [Discussion](https://github.com/confluentinc/ksql/pull/8550) | 
 | [KLIP-60: Support input topics with multiple message types](klip-60-multiple-message-types.md)                                                                             | Proposal |                   |            |  | 
 | [KLIP-61: Bootstrapped TABLE subscriptions](klip-61-bootstrapped-subscriptions.md)                                                     | Proposal |                   |            |  | 
+| [KLIP-62: ksqlDB Ruby Client](klip-62-ksqldb-ruby-client.md)                                                     | Proposal |                   |            | [Discussion](https://github.com/confluentinc/ksql/pull/8794) | 
```

Comment:
we can remove this file from this PR since the intent for klip has been merged

Reply:
@cprasad1 Sure, I've just pushed the commit.